fuel/converters/youtube_audio.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import h5py
+import scipy.io.wavfile
+
+from fuel.converters.base import fill_hdf5_file
+
+
+def convert_youtube_audio(directory, output_directory, youtube_id, channels,
+                          sample, output_filename=None):
+    input_file = os.path.join(directory, '{}.m4a'.format(youtube_id))
+    wav_filename = '{}.wav'.format(youtube_id)
+    wav_file = os.path.join(directory, wav_filename)
+    command = "ffmpeg -y -i {} -ac {} -ar {} {}".format(
```

Comment:
Kinda brittle, but I remember pyffmpeg being a nightmare, so. Any way we can tighten this up a bit?